README.md

```diff
@@ -17,20 +18,22 @@ apt install git python3-dev python3-pip
 
 Clone repository
 ```sh
-git clone git@github.com:kevinjycui/DesmosBezierRenderer.git
+git clone https://github.com/kevinjycui/DesmosBezierRenderer.git
 cd DesmosBezierRenderer
 ```
 
 Install requirements
 ```sh
 python3 -m venv env
 . env/bin/activate
+```
+To set up pypotrace, follow the guide for your OS [here](https://pypi.org/project/pypotrace) *once again, this will not work on windows, use wsl if you are on windows.* Then run:
```

Comment:
Added the potrace dependencies in the Install dependencies section of the setup guide. This line should not longer be needed.